File: app/src/main/java/com/script/Bindings.java
/*
 * Decompiled with CFR 0.152.
 */
package com.script;

import java.util.Map;

public interface Bindings
        extends Map<String, Object> {
    @Override
    public boolean containsKey(Object var1);

    @Override
    public Object get(Object var1);

    @Override
    public Object put(String var1, Object var2);

    @Override
    public void putAll(Map<? extends String, ? extends Object> var1);

    @Override
    public Object remove(Object var1);
}

File: modules/rhino1.3/src/main/java/com/script/Bindings.kt
/*
 * Decompiled with CFR 0.152.
 */
package com.script

interface Bindings : MutableMap<String, Any?> {

    override fun containsKey(key: String): Boolean

    override operator fun get(key: String): Any?

    override fun put(key: String, value: Any?): Any?

    override fun putAll(from: Map<out String, *>)

    override fun remove(key: String): Any?

}